composer.json

```diff
@@ -1,19 +1,38 @@
 {
     "name": "simplethings/entity-audit-bundle",
-    "license": "New BSD",
-    "keywords": ["Persistence", "Database", "Audit"],
+    "license": "LGPL-2.1",
+    "keywords": [
+        "Persistence",
+        "Database",
+        "Audit"
+    ],
     "description": "Audit for Doctrine Entities",
     "require": {
+        "php": "^5.3.9",
         "doctrine/dbal": "~2.5",
-        "doctrine/orm": "~2.2,<2.5"
+        "doctrine/orm": "~2.4"
     },
     "require-dev": {
+        "doctrine/doctrine-bundle": "~1.4",
         "symfony/framework-bundle": "~2.3",
```

Comment:
I will create an PR in the next few days as some deprecated features are still be used.